### ISSUE TITLE
[Feature] Faster Image Load when selecting Performer Image

### ISSUE THREAD

**Issue by iampabber** [stashapp/stash#2683]
**Is your feature request related to a problem? Please describe.**
When scraping and selecting the performer image, I use the left and right arrows to view the available images.

When I click the "next" arrow, I see a "Loading" image, while the real image loads. This adds a delay and prevents me from viewing all images, and selecting one quickly.

**Describe the solution you'd like**
I'd like all the available images to be pre-fetched, so that I don't need to wait for image loads as I scroll through the performer album.

**Describe alternatives you've considered**
None

**Additional context**
![loading_image](https://user-images.githubusercontent.com/107663471/174680257-ab6dc92e-6eb0-4d07-9b4a-6b0871084727.png)

**Comment by DogmaDragon** [comment]
Is this really a problem? It loads instantly for me. I would think having to wait for it to pre-fetch the data would be even more annoying than it loading them on the fly.

**Comment by Gykes** [comment]
> Is this really a problem? It loads instantly for me. I would think having to wait for it to pre-fetch the data would be even more annoying than it loading them on the fly.

Agreed on this, wait on the pre fetch, wait on each item, it's the same amount of time regardless. 

I think loading each image is better for those with slow internet as, at the very least, they can get the first image and move on. Those with faster internet shouldn't have an issue loading them all one at a time.
